docs/src/assets/scss/no-js.scss

```diff
@@ -0,0 +1,67 @@
+@media (prefers-color-scheme: dark) {
+    pre[class*="language-"] {
+        color: var(--color-neutral-100);
+    }
+
+    .token {
+        &.comment,
+        .prolog,
+        .doctype,
+        .cdata {
+            color: #8e9fae;
+        }
+    }
+
+    .logo-component {
+        fill: #fff;
+    }
+
+    #logo-center {
+        opacity: 0.6;
+    }
+
+    .rule__categories,
+    .rule__status {
+        background: var(--body-background-color);
+    }
+
+    .algolia-docsearch-suggestion--highlight {
+        background-color: var(--link-color);
+        color: var(--color-neutral-900);
+    }
+
+    .alert {
+        &.alert--warning {
+            color: var(--color-rose-300);
+            background-color: var(--color-rose-900);
+            border: 1px solid var(--color-rose-300);
+        }
+
+        &.alert--important {
+            color: var(--color-warning-300);
+            background-color: var(--color-warning-900);
+            border: 1px solid var(--color-warning-300);
+        }
+
+        &.alert--tip {
+            color: var(--color-success-300);
+            background-color: var(--color-success-900);
+            border: 1px solid var(--color-success-300);
+        }
+    }
+
+    .alert__type,
+    .alert__learn-more {
+        .alert--warning & {
+            color: var(--color-rose-200);
+        }
+
+        .alert--important & {
+            color: var(--color-warning-200);
+        }
+
+        .alert--tip & {
+            color: var(--color-success-200);
+        }
+    }
+}
```

Comment:
```suggestion
@media (prefers-color-scheme: dark) {
    :not([data-theme="light"]) {
        pre[class*="language-"] {
            color: var(--color-neutral-100);
        }

        .token {
            &.comment,
            .prolog,
            .doctype,
            .cdata {
                color: #8e9fae;
            }
        }

        .logo-component {
            fill: #fff;
        }

        #logo-center {
            opacity: 0.6;
        }

        .rule__categories,
        .rule__status {
            background: var(--body-background-color);
        }

        .algolia-docsearch-suggestion--highlight {
            background-color: var(--link-color);
            color: var(--color-neutral-900);
        }

        .alert {
            &.alert--warning {
                color: var(--color-rose-300);
                background-color: var(--color-rose-900);
                border: 1px solid var(--color-rose-300);
            }

            &.alert--important {
                color: var(--color-warning-300);
                background-color: var(--color-warning-900);
                border: 1px solid var(--color-warning-300);
            }

            &.alert--tip {
                color: var(--color-success-300);
                background-color: var(--color-success-900);
                border: 1px solid var(--color-success-300);
            }
        }

        .alert__type,
        .alert__learn-more {
            .alert--warning & {
                color: var(--color-rose-200);
            }

            .alert--important & {
                color: var(--color-warning-200);
            }

            .alert--tip & {
                color: var(--color-success-200);
            }
        }
    }
}
```

Reply:
We can wrap it inside the `no-js` class. It will fix the issue I guess.

Reply:
> We can wrap it inside the `no-js` class. It will fix the issue I guess.

Sounds good to me. @kecrily what do you think?

Reply:
Sounds good for me